clients/cloud/java/pom.xml

```diff
@@ -35,9 +35,9 @@
 
     <properties>
         <java.version>1.8</java.version>
-        <kafka.version>2.0.2-SNAPSHOT</kafka.version>
+        <kafka.version>2.1.0</kafka.version>
         <kafka.scala.version>2.11</kafka.scala.version>
-        <confluent.version>5.0.2-SNAPSHOT</confluent.version>
+        <confluent.version>5.0.1</confluent.version>
```

Comment:
How will we remember to update these as 5.1 is released?

Reply:
DevX is responsible for updating the release for the entire `examples` repo.